h/features.py

```diff
@@ -118,13 +130,36 @@ def enabled(self, name):
         If the feature has no override in the database, it will default to
         False. Features must be documented, and an UnknownFeatureError will be
         thrown if an undocumented feature is interrogated.
+
+        When the internal cache is empty, it will automatically load the
+        feature flags from the database first.
         """
         if name not in FEATURES:
             raise UnknownFeatureError(
                 '{0} is not a valid feature name'.format(name))
 
-        feature = self._cache.get(name)
+        if not self._cache:
+            self.load()
+
+        return self._cache.get(name, False)
```

Comment:
I think at this point this should be a `self._cache[name]`, no? If `name` is unknown we just raised an `UnknownFeatureError`, and `self.load` creates entries for everything in `FEATURES`.